### PR TITLE
Minor tweaks to Sri's text

### PR DESCRIPTION
**Pull request by boucadair**

### CODE REVIEW

File: draft-brw-sconepro-rate-policy-discovery.md [hunk 1]
```diff
@@ -412,7 +412,7 @@ The format of the IPv6 RA NRLP option, with optional fields included, is illustr
  0                   1                   2                   3
  0 1 2 3 4 5 6 7 8 9 0 1 2 3 4 5 6 7 8 9 0 1 2 3 4 5 6 7 8 9 0 1
 +-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+
-|     Type      |     Length    |D|E|P|R| Scope |      TC       |
+|     Type      |     Length    |D|E|P|R|U|U| Scope |    TC     |
```
Comment (boucadair) [inline draft-brw-sconepro-rate-policy-discovery.md:415]: @sridharangirish91: I wonder whether it makes more sense to define "R" as a TC value.

Comment (sridharangirish91) [reply draft-brw-sconepro-rate-policy-discovery.md:415]: Hey Med,

I initially thought of that but things in TC seems to be nature of the application and data but the reliability is more of a stream, which is broader nature.

For eg. Real-time traffic can be both reliable (keyboard and mouse input) and unreliable (video P frames). 

That's why kept a separate flag which can be treated globally above TC.

Comment (boucadair) [reply draft-brw-sconepro-rate-policy-discovery.md:415]: We actually need to separate 
* flags to control data that is enclosed in the options
vs
* characteristics of the flow themselves.

I will merge this PR, but will think about how to better articulate this in the spec.

Comment (sridharangirish91) [reply draft-brw-sconepro-rate-policy-discovery.md:415]: Thanks Med

Comment (boucadair) [reply draft-brw-sconepro-rate-policy-discovery.md:415]: See #17

**Pull request merged**